doc/userguide/rules/dns-keywords.rst

```diff
@@ -46,6 +46,29 @@ Match on DNS requests where the **opcode** is NOT 0::
 
   dns.opcode:!0;
 
+dns.rrtype
+----------
+
+This keyword matches on the **rrtype** found in the DNS message.
+
```

Comment:
Could you follow the style where you indicate that this is an integer keyword, and a reference to the integer keyword documentation that was recently merged? See example here: https://github.com/OISF/suricata/blob/master/doc/userguide/rules/dhcp-keywords.rst#dhcprebinding_time